bases/next.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Next.js",
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": false,
```

Comment:
can I ask why "strict" => false; it seems a good practice to enable it?

Reply:
That's what next folks do, it's there call but it's not unreasonable to me - not every project needs to be in strict mode

Reply:
Strict mode was turned off in favor of people who might not be comfortable with TypeScript but still want to have the best IntelliSense experience. In their documentation, they actually recommend to turn on strict mode when comfortable.

![frenco's screenshot](https://user-images.githubusercontent.com/17830687/182202531-adc8d803-8d58-4009-ac8d-efd27c2f646c.png)

I think there should also be a next + strictest option. What do you think @orta?

Reply:
I think we're fine without, if they default to not strict then having strict on is a _project_ specific change and making it easy to see that choice in the `tsconfig.json` is a win. If it came from here, people might think its a next-js made call.

Reply:
Yea that makes sense, but I went ahead and made a PR. You can merge it if you want, or close it if you will.